.github/workflows/pr_test_cpu.yml

```diff
@@ -0,0 +1,56 @@
+name: Run tests on CPU
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+    types: [open, reopened, synchronize, review_requested, ready_for_review]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  tests-cpu:
+    name: ${{ matrix.os }} - python-${{ matrix.python-version }}, torch-${{ matrix.pytorch-version }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos]
+        python-version: ['3.7', '3.8']
+        pytorch-version: ['1.10.2', '1.12.1']
```

Comment:
Maybe update to 1.9 and 1.13 for pr? - min/max version

Reply:
i think yes @ducha-aiki

Reply:
done on [bdeab40](https://github.com/kornia/kornia/pull/2009/commits/bdeab40f22bcfbbda43d233570f96c33aa6dab6c)

Reply:
Seems some fixes need to be done for 1.13.  Maybe it is better to keep 1.12.1, and do this migration in a subsequent PR, what do you think?